### ISSUE TITLE
[CST V2] Remove "sighted language" for 1P evidence requests

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#80625]
## Background Context

We got [feedback in the V2 Midpoint Review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64935) that we should remove "sighted language" on evidence request alerts. 

So instead of "View details" it should be "Details"

## Design

Replace "View details" with "Details" on 1P request card

Before
<img width="700" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUJUQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0c0a77bb0b44a4e87b507c2fe68a6ab727424ed2/Screenshot%202024-04-11%20at%201.24.54%E2%80%AFPM.png" alt="Screenshot 2024-04-11 at 1.24.54 PM.png" />

After
<img width="1744" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBODlUQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a0882c0f5fdf2c497a105fa1a2a5a8c5fd63710b/Screenshot%202024-04-11%20at%201.22.01%E2%80%AFPM.png" alt="Screenshot 2024-04-11 at 1.22.01 PM.png" />

## Engineering 
- We are only updating the V2 FilesNeeded.jsx component that is shown when the toggle cst_use_claim_details_v2 is enabled

## Out of scope
- FilesNeededOld. jsx and FilesOptionalOld.jsx that have buttons that say 'View details' 
![Screenshot 2024-04-25 at 4 06 03 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/33a91057-953d-499f-8e52-571c4242083f)

## Open questions
- Add here

## Tasks
- [ ] _Update link language on FilesNeeded.jsx 
- [ ] Update link aria text on FilesNeeded.jsx 

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] The First party alert (P1 alert) on the status and files tab no longer has a link that says 'View details' and has a link that says 'Details'
- [ ] The First party alert (P1 alert) on the status and files tab no longer has aria text that says 'View details' and instead references just 'details'

**Comment by pmclaren19** [comment]
Talked to @skylerschain to confirm a few things:

- We are only updating the V2 `FilesNeeded.jsx` component that is shown when the toggle `cst_use_claim_details_v2` is enabled
- There is aria text that is shown 
![Screenshot 2024-04-25 at 3 57 50 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/7861c115-07ab-4d40-98f6-1e72565bbac5)
Confirming with @davidakennedy that we want to also update this to no longer include the 'view details text and to just say 'details'

Updating the ticket info and adding acceptance criteria with this.

**Comment by davidakennedy** [comment]
@skylerschain and @pmclaren19 Thanks for double checking this.

Let's have the `aria-label` read out: "Details for ${item.displayName}`. This way, it's closer to the visual text, which is important for [Label in Name, a WCAG criteria](https://www.w3.org/TR/WCAG22/#label-in-name).

**Comment by pmclaren19** [comment]
Verified in Staging and looks good! Closing!
